docs/installation.md

```diff
@@ -18,8 +18,8 @@ pipx ensurepath
 Otherwise, install via pip (requires pip 19.0 or later):
 
 ```
-python3 -m pip install --user pipx
-python3 -m pipx ensurepath
+py -3 -m pip install --user pipx
+py -3 -m pipx ensurepath
 ```
```

Comment:
This block is describing not just Windows but other platforms as well. You need to add a new paragraph insterad.

Reply:
Thanks, I've added a new paragraph.